x/slashrefund/keeper/msg_server.go

```diff
@@ -94,6 +94,12 @@ func (ms msgServer) Withdraw(goCtx context.Context, msg *types.MsgWithdraw) (*ty
 		return nil, valErr
 	}
 
+	// Check if validatorAddress corresponds to a validator
```

Comment:
I think we should allow to withdraw using an account that now is no more associated to a validator.

Reply:
I agree. If validator has been removed, then deposited funds or generated refunds linked to this validator would be unreachable. We should update this logic. I suggest doing it with a different pull request.